packages/extension/src/components/pages/Login/Login.tsx

```diff
@@ -154,9 +176,17 @@ export const Login: FC = () => {
           });
       }
     } else {
-      setPasswordError('Incorrect password');
+      if (currentAttempts >= maxAttempts) {
+        let TIMESTAMP = Date.now();
```

Comment:
Timestamp is never re-assigned please use const instead of let.

Reply:
done in 1ac1bd390fd0d24cd765dafe475216c48b573ce8